RecoTracker/LSTCore/src/classes_def.xml

```diff
@@ -0,0 +1,18 @@
+<lcgdict>
+  <class name="lst::Params_pLS::ArrayUxHits"/>
```

Comment:
Since this type is really `edm::StdArray<unsigned int, 4>`, it should be placed in `DataFormats/Common/src/classes_def.xml` (either in this or in a separate PR).